client/dist/styles/related_lists.css

```diff
@@ -25,6 +25,11 @@
   background: lightgrey;;
   width: 100%;
   height: 323px;
```

Comment:
Don't forget Leslie's recommendations for not hard coding pixels but using em units since these scale with different display sizes.